app/assets/javascripts/discourse/app/components/modal/bulk-topic-actions.gjs

```diff
@@ -1,19 +1,35 @@
 import Component from "@glimmer/component";
-import { action } from "@ember/object";
+import { tracked } from "@glimmer/tracking";
+import { action, computed } from "@ember/object";
+import { empty } from "@ember/object/computed";
 import { inject as service } from "@ember/service";
 import { Promise } from "rsvp";
+import ChangeTags from "discourse/components/bulk-actions/change-tags";
 import DButton from "discourse/components/d-button";
 import DModal from "discourse/components/d-modal";
+import RadioButton from "discourse/components/radio-button";
+import { topicLevels } from "discourse/lib/notification-levels";
 import Topic from "discourse/models/topic";
 import htmlSafe from "discourse-common/helpers/html-safe";
 import i18n from "discourse-common/helpers/i18n";
-//import AppendTags from "../bulk-actions/append-tags";
-//import ChangeCategory from "../bulk-actions/change-category";
-//import ChangeTags from "../bulk-actions/change-tags";
-//import NotificationLevel from "../bulk-actions/notification-level";
+import CategoryChooser from "select-kit/components/category-chooser";
+import TagChooser from "select-kit/components/tag-chooser";
 
 export default class BulkTopicActions extends Component {
   @service router;
+  @tracked activeComponent = null;
+  @tracked tags = [];
+
+  categoryId = 0;
+  notificationLevelId = null;
+  @empty("notificationLevelId") disabled;
```

Comment:
Can you expand on what this does? I can't find where `disabled` is used

Reply:
I don't remember why I added that, let's remove it!